IdentityCore/src/webview/response/MSIDCBAWebAADAuthResponse.m

```diff
@@ -58,6 +58,7 @@ - (instancetype)initWithURL:(NSURL *)url
     {
         NSURLComponents *resultUrlComponents = [NSURLComponents componentsWithURL:url resolvingAgainstBaseURL:NO];
         resultUrlComponents.query = nil;
+        resultUrlComponents.fragment = nil;
```

Comment:
From [spec](https://tools.ietf.org/html/rfc6749#section-3.1.2), redirect uri should not contain fragment. Do we do this just for CBA response?

Reply:
Good point, added validation to URI provided by the developer as well.